wikipedia_cleanup/data_filter.py

```diff
@@ -86,6 +86,75 @@ def __str__(self) -> str:
         )
 
 
+class DiscardAttributesDataFilter(AbstractDataFilter):
+    def _filter_for_property(self, changes: List[InfoboxChange]) -> List[InfoboxChange]:
+        raise NotImplementedError("This method should never be called.")
+
+    def filter(
+        self, changes: List[InfoboxChange], initial_num_changes: int
+    ) -> List[InfoboxChange]:
+        self.filter_stats.initial_num_changes = initial_num_changes
+        self.filter_stats.input_num_changes = initial_num_changes
+        self.filter_stats.output_num_changes = initial_num_changes
+
+        sparse_changes = []
+        for change in changes:
```

Comment:
I'm actually not sure how you filter changes here, can you explain?

Reply:
You only filter the attributes of each change. All changes are preserved but with fewer attributes